### PR TITLE
Very minor edits to Alice's Abuse

### PR DESCRIPTION
**Pull request by shannona**

### CODE REVIEW

File: final-documents/alice-attempts-abuse-verifiable-credential.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Alice searches the dark web, looking for buyers of her prescription credential.
 But Alice has much less success than she expected. It turns out that her credential is mostly worthless on the black market, for one or more of the following reasons:
 
 #### Traditionally-signed Credential
-* The credential is bound to a DID, and credential holders are required to disclose the same DID every time they fill a prescription. The verifier can check to see if a person purporting to be Alice is using the same DID with this prescription as they used last week with a different prescription, and reject the credential if there is a mismatch.
+* The credential is bound to a DID, and credential holders are required to disclose the same DID every time they fill a prescription. The verifier can check to see if a person purporting to be Alice is using the same DID with this prescription as they used last week with a different prescription and reject the credential if there is a mismatch.
```
Comment (dhh1128) [inline final-documents/alice-attempts-abuse-verifiable-credential.md:71]: This change introduces a run-on sentence.

File: final-documents/alice-attempts-abuse-verifiable-credential.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ For simplicity it is assumed that the system consists of three categories of par
 
 ### Alice tampers with the credential 
 
-To start our scenario, Alice receives a valid prescription for melange from her doctor’s office, in the form of a verifiable credential. However, she is not happy with the dosage. She decides to tamper with the credential and increase the amount of melange that she can buy. She changes the dosage text value within the credential from 250mg to 500mg and presents the modified credential to the pharmacy.
+To start our scenario, Alice receives a valid prescription for melange from her doctor’s office in the form of a verifiable credential. However, she is not happy with the dosage. She decides to tamper with the credential and increase the amount of melange that she can buy. She changes the dosage text value within the credential from 250mg to 500mg and presents the modified credential to the pharmacy.
```
Comment (dhh1128) [inline final-documents/alice-attempts-abuse-verifiable-credential.md:26]: This change makes the sentence ambiguous; what is int he form of a verifiable credential--the doctor's office or the prescription?

**Review by dhh1128 — APPROVED** [review]
Fine. Note, however, that I think a couple of the comma deletions are a mistake. Might be a difference in editorial preference.

File: final-documents/alice-attempts-abuse-verifiable-credential.md (hunk 1 above)
Comment (shannona) [reply final-documents/alice-attempts-abuse-verifiable-credential.md:71]: The standard rule here is that if two verbs have the same subject, there is not a comma. I'm going to change "reject" to "can reject" to clarify the parallel.

File: final-documents/alice-attempts-abuse-verifiable-credential.md (hunk 2 above)
Comment (shannona) [reply final-documents/alice-attempts-abuse-verifiable-credential.md:26]: I'm reverting that, per your suggestion.

**Pull request merged**